src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/EmbeddedLanguages/VirtualChars/CSharpVirtualCharService.cs

```diff
@@ -111,7 +123,9 @@ private static VirtualCharSequence TryConvertVerbatimStringToVirtualChars(Syntax
             if (skipDelimiterQuotes)
             {
                 Contract.ThrowIfFalse(tokenText[0] == '"');
-                Contract.ThrowIfFalse(tokenText[^1] == '"');
+
+                if (tokenText.Length >= 2 && tokenText[endIndexExclusive - 1] == '8' && tokenText[endIndexExclusive - 2] is 'u' or 'U')
```

Comment:
Is there a syntax token that represents the 'u'/'U' or is the entire string a string token or utf8 string token, or do we need to actually look at the text of it?  (am not too familiar with this system)

Or could we say if the token is a utf8 string, remove 2 from the end

Reply:
sgtm.  changd to do that.